packages/twenty-server/src/core/messaging/timeline-thread-participant.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+
+import { Entity } from 'typeorm';
+
+@Entity({ name: 'timelineThreadParticipant', schema: 'core' })
```

Comment:
We don't need this since this is not a typeorm entity (there is no table for that object)